XenModel/Actions/NRPE/NRPEUpdateAction.cs

```diff
@@ -104,6 +104,23 @@ private void SetNRPEConfigureForHost(IXenObject o)
 
         private void SetNRPEStatus(IXenObject host, bool enableNRPE)
         {
+            if (enableNRPE)
+            {
+                Dictionary<string, string> enableDict = new Dictionary<string, string>
+                {
+                    { "enable", "true" }
+                };
+                try
+                {
+                    string controlResult = Host.call_plugin(host.Connection.Session, host.opaque_ref, NRPEHostConfiguration.XAPI_NRPE_PLUGIN_NAME,

```

Comment:
I can see future devs wondering why this was added. Let's make their lives easier by adding a comment here with the ticket number to tell them this is needed without having to `git blame`.

something like

```
// CA-384579: If the certificate used by NRPE was generated in Yangtze it has no permissions. 
// We prevent command (start, restart, etc) failures by calling control, which modifies certificate permissions.
```

Reply:
Added it.

Reply:
wasn't aware that we were planning to remove it in the future! maybe we didn't need the comment, then. but doesn't hurt. ty!